rmw_zenoh_cpp/src/detail/rmw_context_impl_s.cpp

```diff
@@ -67,7 +69,37 @@ class rmw_context_impl_s::Data final : public std::enable_shared_from_this<Data>
     if (!config.has_value()) {
       throw std::runtime_error("Error configuring Zenoh session.");
     }
-
+#ifdef HAVE_SECURITY
```

Comment:
Why do we need this flag? My understanding is that `rmw_dds_common::get_security_files` will return false if the enclave ROS arg is not passed or is invalid so we can have this codeblock that modifies for all situations. Let me know if I misunderstood something.